source/opt/instruction.h

```diff
@@ -592,6 +596,10 @@ class Instruction : public utils::IntrusiveNodeBase<Instruction> {
   // DebugScope that wraps this instruction.
   DebugScope dbg_scope_;
 
+  // True if it is an OpLine instruction that does not exist in the input, but
+  // injected by IrLoader to help tracking the line info during transformations.
+  bool is_extra_line_;
```

Comment:
I'm not sure we should be adding an extra member variable to an instruction just for this.

Reply:
Right.